### DISCUSSION TITLE
forcing 4:3 fov in 16:9 resolution

### DISCUSSION THREAD

**Discussion by ghost** [hzqst/MetaHookSv#322] [General]
hello hzqst i really like the r_adjust_fov 2 feature in the release but is it possible that you could make the same plugin for the default cs 1.6

**Comment by hzqst** [comment]
Forgot to mention that it has already been implemented for both SvEngine and GoldSrc: https://github.com/hzqst/MetaHookSv/blob/main/docs/Renderer.md#widescreen-adapted-fov
